<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MSI" failures="0" tests="3" name="mystepdefs.signup" time="0.014" errors="1" timestamp="2024-02-29T14:35:45 IST" skipped="2">
  <testcase classname="mystepdefs.signup" name="i_should_see_an_error_message_indicating_please_match_the_requested_format" time="0.011">
    <skipped/>
  </testcase> <!-- i_should_see_an_error_message_indicating_please_match_the_requested_format -->
  <system-out/>
  <testcase classname="mystepdefs.signup" name="i_should_see_an_error_message_indicating_please_match_the_requested_format" time="0.002">
    <skipped/>
  </testcase> <!-- i_should_see_an_error_message_indicating_please_match_the_requested_format -->
  <system-out/>
  <testcase classname="mystepdefs.signup" name="i_should_see_an_error_message_indicating_please_match_the_requested_format" time="0.001">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.fail(String)&quot; because &quot;mystepdefs.signup.test&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.fail(String)" because "mystepdefs.signup.test" is null
at mystepdefs.signup.i_should_see_an_error_message_indicating_please_match_the_requested_format(signup.java:115)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- i_should_see_an_error_message_indicating_please_match_the_requested_format -->
  <system-out/>
</testsuite> <!-- mystepdefs.signup -->
